### ISSUE TITLE
the bad result on rect mask

### ISSUE THREAD

**Issue by ljjcoder** [shepnerd/inpainting_gmcnn#38]
@shepnerd thanks for your excellent work！I notice you release the stroke model on places2，it works well on stoke mask。 but when I finetune the model with rect mask from your released model， the test result often occurs heavily artifacts。for example：
 
![1571371357(1)](https://user-images.githubusercontent.com/40750189/67065006-39016e80-f19f-11e9-94fc-7da030684f44.png)
![1571371580(1)](https://user-images.githubusercontent.com/40750189/67065134-baf19780-f19f-11e9-95d3-4befa51edce6.png)
![1571371621(1)](https://user-images.githubusercontent.com/40750189/67065161-d2c91b80-f19f-11e9-878e-aaf15eab57d9.png)
is it a normal result with rect mask on places2? Can you release your trained model on places2 with rect mask?

**Comment by shepnerd** [comment]
The pretraining stage of using rect masks differs a lot from that using stroke masks (the former needs to predict more areas than the later), so it will cause such artifacts. You can email me for the model.

**Comment by izillio** [comment]
Hello, I have the same problem. Can you please share the weights with me?
@shepnerd  @ljjcoder
